README.md

```diff
@@ -245,12 +245,21 @@ labelled as active.
 2. Create a new key
 3. Start the key rotation cycle
 
-#### Key rotation
+#### Key renewal
 
-Keys are automatically rotated. This can be configured on controller startup with
+Keys are automatically renewed. This can be configured on controller startup with
 the `--rotate-period=<value>` flag. The `value` field can be given as golang
 duration flag (eg: `720h30m`).
 
+The feature has been historically called "key rotation" but this term can be confusing.
+Sealed secrets are not automatically rotated and old keys are not deleted
+when new keys are generated. Old sealed secrets resources can be still decrypted.
+
+If you want to rotate the sealed secrets themselves (i.e. re-encrypt them with the latest private key),
+you need to invoke `kubeseal --rotate` which will produce a new sealed secret file freshly encrypted with
+the latest key, without making the secrets leave the cluster to the client. You can then save that file
+in your version control system (`kubeseal --rotate` doesn't update the in-cluster object).
+
```

Comment:
please provide a cli cmds example, as it's a bit confusing to read `kubeseal --rotate` verbatim (without `-n NAMESPACE NAME` or alike) that IMO reads as general/cluster-wide operation to be performed, rather than an outputted sealedsecret content (as it becomes clearer near the end of the paragraph).

Reply:
good catch; done